llvm/lib/SYCLLowerIR/CompileTimeProperties.def

```diff
@@ -54,3 +54,9 @@ SYCL_COMPILE_TIME_PROPERTY("sycl-stride-size", 5883, DecorValueTy::uint32)
 SYCL_COMPILE_TIME_PROPERTY("sycl-word-size", 5884, DecorValueTy::uint32)
 SYCL_COMPILE_TIME_PROPERTY("sycl-bi-directional-ports-true", 5885,
                            DecorValueTy::none)
+
+// The corresponding SPIR-V OpCodes for sycl_ext_oneapi_prefetch
+// SYCL Spec: https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_prefetch.asciidoc
+// SPIR-V Spec: https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/INTEL/SPV_INTEL_cache_controls.asciidoc
+SYCL_COMPILE_TIME_PROPERTY("sycl-prefetch-hint", 6443, DecorValueTy::uint32)
```

Comment:
I see in the extensions doc that 6443 corresponds to CacheControlStoreINTEL.
I was thinking that prefetch should correspond to a load (6442)?

Thanks

Reply:
> Is it a store because it 'stores' into a cache?

Yep, I thought that prefetch is something like Store instruction. But, since the prefetch instruction has `Result<id>`, it's more like OpLoad. So I'll change it.